core/chains/evm/txmgr/eth_resender_test.go

```diff
@@ -140,7 +140,7 @@ func Test_EthResender_Start(t *testing.T) {
 			er.Start()
 			defer er.Stop()
 
-			cltest.EventuallyExpectationsMet(t, ethClient, 5*time.Second, 10*time.Millisecond)
+			cltest.EventuallyExpectationsMet(t, ethClient, 5*time.Second, time.Second)
```

Comment:
This already runs in parallel and has an overall timeout, so a longer interval doesn't cost us much of anything here.
Also, not a problem yet, but we could derive the timeout from the test deadline instead of fixed at 5s too.